pages/_template.jsx

```diff
@@ -2,31 +2,109 @@ import React, { PropTypes } from 'react';
 import Headroom from 'react-headroom';
 import { Link } from 'react-router';
 import { prefixLink } from 'gatsby-helpers';
+import classNames from 'classNames';
```

Comment:
The module is `classnames`, idk why this works for you (it didn't for me) but when i changed it to `classnames` the hamburger thing works fine